diff --git a/main.py b/main.py
--- a/main.py
+++ b/main.py
@@ -197,6 +197,11 @@ if __name__ == "__main__":
         except:
             pass
 
+    if args.user_directory:
+        user_dir = os.path.abspath(args.user_directory)
+        logging.info(f"Setting user directory to: {user_dir}")
+        folder_paths.set_user_directory(user_dir)
+
     loop = asyncio.new_event_loop()
     asyncio.set_event_loop(loop)
     server = server.PromptServer(loop)
